doc/releases/changelog-0.25.0.md

```diff
@@ -565,6 +565,69 @@ of operators.
       params, cost = opt.step_and_cost(cost, params)
   ```  
 
+<h4>Experimental feature: New return types</h4>
+
+* We provide a new experimental return types system for QNodes with multiple measurements.
+  
+  [(#2814)](https://github.com/PennyLaneAI/pennylane/pull/2814)
+  [(#2815)](https://github.com/PennyLaneAI/pennylane/pull/2815)
+  [(#2860)](https://github.com/PennyLaneAI/pennylane/pull/2860)
+  
+  The new system guarantees an intuitive return types, it is returning tuple of measurements by default. It is now
+  possible to trigger the new return type system with the top-level function `qml.enable_return`. We have support for
+  forward pass and backpropagation for every interface. It does not support custom gradient function for now.
+  
+  **Example**
+  
+  Forward pass:
+  ```python
+  qml.enable_return()
+  
+  dev = qml.device("default.qubit", wires=2)
+  
+  def circuit(x):
+          qml.Hadamard(wires=[0])
+          qml.CRX(x, wires=[0, 1])
+          return (qml.probs(wires=[0]), qml.vn_entropy(wires=[0]), qml.probs(wires=0), qml.expval(wires=1))
+  
+  qnode = qml.QNode(circuit, dev)
+  ```
+  ```pycon
+  res = qnode(0.5)
+  >>> res
+  (tensor([0.5, 0.5], requires_grad=True), tensor(0.08014815, requires_grad=True), tensor([0.5, 0.5], requires_grad=True), tensor(0.93879128, requires_grad=True))
+  ```
```

Comment:
```suggestion

Access this new feature by calling the `qml.enable_return()` function and in a subsequently-defined QNode, pack your different measurement types into a tuple:

  ```python
  qml.enable_return()
  
  dev = qml.device("default.qubit", wires=2)
  
  @qml.qnode(dev)
  def circuit(x):
          qml.Hadamard(wires=[0])
          qml.CRX(x, wires=[0, 1])
          return (qml.probs(wires=[0]), qml.vn_entropy(wires=[0]), qml.probs(wires=0), qml.expval(wires=1))
  ```
  ```pycon
  >>> circuit(0.5)
  (tensor([0.5, 0.5], requires_grad=True), tensor(0.08014815, requires_grad=True), tensor([0.5, 0.5], requires_grad=True), tensor(0.93879128, requires_grad=True))
  ```
```

Reply:
Here it is rendering strange in GitHub, do you suggest to remove the backprop example.

Reply:
Yeah I'm not married to the backpropagation example. Simply stating "In addition, QNodes that utilize this new return type support backpropagation."

Reply:
Ok no worries let's remove it!

Reply:
Sorry is there a reason why this suggestion wasn't committed?